lib/mappings/sendReceive.js

```diff
@@ -1,25 +1,57 @@
 import {
   selectWallets,
+  selectMultisigWallets,
   selectWalletAccounts,
   selectSelectedWallet,
   selectSelectedAccount,
+  selectWalletAccountInfo,
 } from './selectors';
 
-import { selectWallet, getWallets, selectAccount } from '../actions';
+import {
+  selectWallet,
+  getWallets,
+  selectAccount,
+  joinWallet,
+  sendTX,
+  createAddress,
+} from '../actions';
+
+/*
+ * TODO: parse receive address and pass into container
+ */
 
 function mapStateToProps(state, otherProps) {
   const wallets = selectWallets(state);
+  const multisigWallets = selectMultisigWallets(state);
   const selectedWallet = selectSelectedWallet(state);
   const selectedAccount = selectSelectedAccount(state);
-  let accounts = selectWalletAccounts(state, selectedWallet);
 
+  // TODO: handle multisig/standard
+  const accountInfo = selectWalletAccountInfo(state);
+  let receiveAddress, selectedAccountInfo;
+  try {
+    selectedAccountInfo = accountInfo[selectedWallet][selectedAccount];
+    receiveAddress = selectedAccountInfo.receiveAddress;
+  } catch (e) {}
+
+  let accounts = selectWalletAccounts(state, selectedWallet);
   if (!accounts) accounts = [];
 
+  // watch out for same wallet being included twice
```

Comment:
this doesn't account for a duplicate wallet right now right? since it just pushes each one on the array, a duplicate could get pushed from both right?